wrappers/Mathematica/RefpropLink/Kernel/RefpropLink.wl

```diff
@@ -224,30 +226,34 @@ Clear[$dllPath,$fluidsPath,$mixturesCoeffPath,$fluidmixComponents];
 (*     (default: C:\Program Files (x86)\REFPROP\REFPRP64.dll                           *)
 (*  2. Parent path above \fluids & \mixtrues directory                                 *)
 (*     (default: C:\Program Files (x86)\REFPROP                                        *)
-(* If the text file exists, use it.  If not, set default directory paths.              *)
+(* If the text file exists, use it.  If not, use $RPprefix location for both.          *)
+(* If $RPprefix is not set in environment set default directory paths.                 *)
 (***************************************************************************************)
 $filepath = "";                                              (* Set default file paths *)
 $dllPath = "C:\\Program Files (x86)\\REFPROP\\REFPRP64.dll";
 $NISTPath = "C:\\Program Files (x86)\\REFPROP"
-(* Check for FilePaths Resource, which will overwrite these paths                      *)
-$hPaths = PacletManager`PacletResource["RefpropLink","FilePaths"];
-If[$hPaths === Null,                                            (* Can't find Resource *)
-    hEclPaths = FileNameJoin[{$HomeDirectory, "eclipse-workspace", "RefpropLink", "RefpropLink", "TextResources", "Filepaths.txt"}];
-    If[FileExistsQ[hEclPaths],         (* See if there's an Eclipse workspace resource *)
-        $hPaths = hEclPaths;           (*            IF so, set file paths from there. *)
-        (* Print["Using Eclipse-Working Resources. hPaths = ",$hPaths]; *)
-    ];
+
+(* Check for RPPrefix if REFPROP is installed in a non-standard location               *)
+If[!FailureQ[RPPath =Environment["RPprefix"]],
+    (* Print["RPPrefix is set to ",RPPath]; *)
+    $NISTPath = RPPath;
+    $dllPath = FileNameJoin[{$NISTPath,"REFPRP64.dll"}]
```

Comment:
Is this problematic on 32-bit OS or non-windows OS (if such a thing is supported)?